compiler/optimizing/code_generator_vector_x86.cc

```diff
@@ -220,14 +220,23 @@ void InstructionCodeGeneratorX86::VisitVecReduce(HVecReduce* instruction) {
       DCHECK_EQ(4u, instruction->GetVectorLength());
       switch (instruction->GetKind()) {
         case HVecReduce::kSum:
-          __ movaps(dst, src);
+        	if (CpuHasAVXorAVX2FeatureFlag()) {
```

Comment:
As discussed, though you can have the facilities ready for emitting the vmovaps but unneeded emitting them (as in this case) is not required unless you have an advantage of using them.